microsoft/testsuites/kdump/kdumputils.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Move it to tools folder.

Reply:
@squirrelsc I have not created tool out of it. Should we create tool?

Reply:
Yes, please create a tool, don't share tools between different test suites.